src/charm.py

```diff
@@ -77,91 +79,91 @@ def configure_workload_container(self, event) -> None:
         """Configure/set up workload container inside pod."""
         missing = self._missing_charm_configs()
         if missing:
-            msg = 'Required config(s) empty: {}'.format(', '.join(sorted(missing)))
+            msg = "Required config(s) empty: {}".format(", ".join(sorted(missing)))
```

Comment:
Could this be an f-string?
https://github.com/canonical/is-charms-contributing-guide/blob/main/CONTRIBUTING.md#f-strings

Reply:
Surely, I only focused on getting the integration test to work on this PR, but in the next ones I'll take care of all linting/formatting issues (the charm is quite old so it is potentially plagued with this kind of issues)